class.jetpack-admin.php

```diff
@@ -60,12 +60,13 @@ static function sort_requires_connection_last( $module1, $module2 ) {
 	// presentation like description, name, configuration url, etc.
 	function get_modules() {
 		include_once( JETPACK__PLUGIN_DIR . 'modules/module-info.php' );
-		$available_modules = $this->jetpack->get_available_modules();
-		$active_modules    = $this->jetpack->get_active_modules();
+		$get_object_module = $this->jetpack;
```

Comment:
Let's just call the method statically using the class name like it's done everywhere else: `Jetpack::get_available_modules()`. Thank you!